debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -357,6 +357,21 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
             if (mode == null && defaultValue != null) mode = parse(defaultValue);
             return mode;
         }
+
+        public static enum EventDeserializationFailureHandlingMode implements EnumeratedValue {
+
+            IGNORE,
+            WARN,
+            FAIL;
+
+            @Override
+            public String getValue() {
+                return null;
+            }
+
+
+
```

Comment:
Minor: do we need all these blank lines? :-)

Reply:
Nope :) That's just a placeholder, still need to code that option and parsing of the enum. Just wanted to make sure you agree on the meat of the change, i.e. the logic within the binlog reader. 

Thanks a lot for the quick feedback!